src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
+  let fullAmount = 0;
+
+  for (let i = 1; i <= days; i++) {
+    fullAmount += rentDay;
+  }
+
+  if (days >= 7) {
+    fullAmount = fullAmount - 50;
+
+    return fullAmount;
```

Comment:
You're using a literal for the discount value. According to the checklist item 2, use constants for such values to make the code more readable and maintainable. For instance, 'const LONG_TERM_DISCOUNT = 50;'.